data/strings/sound.txt

```diff
@@ -429,6 +429,92 @@
     zh-Hans = 出口.
     zh-Hant = 出口.
 
+  [route_beginning]
+    en = Follow the highlighted route.
```

Comment:
This English (and translations) seems a bit off to me. Apple Maps starts with "Starting route". Google Maps seems to not have this first string, starts with "Head west etc"

Reply:
@matheusgomesms yeah there's no perfect option and it's worth discussing. Can you add your experience to #2802 where I've documented some alternatives? This is just the simplest, easiest to implement option but probably not the best

Reply:
That's what I just did, thanks!

Reply:
Considering OM's simplicity to use, and the availability of some test strings in settings, this announcement is not needed, because the user already knows what he/she just did.

Reply:
@biodranik that's fine, in which case the attached issue should be closed as wontfix and I'll remove "beginning route" as an announcement

Reply:
Announcing the first instruction sounds useful. We can fix/merge this PR for recalculation only, and then implement the first instruction in a separate PR.